docs/tools/tools_intro.md

```diff
@@ -43,38 +43,10 @@ The Mbed Online Compiler is our in-house IDE and should be familiar to anyone wh
 | Arm v7A       | `Arm Compiler 6.11` |
 | Arm v8M       | `Arm Compiler 6.11` |
 
-For more information, please see the [Online Compiler page](developing-mbed-online-compiler.html).
+<span class="notes">**Note:** Arm Compiler 6 is the default Arm Compiler for Mbed OS development. Most Mbed OS platforms are already compatible with Arm Compiler 6. Some existing targets still supporting Arm Compiler 5 will be migrated to Arm Compiler 6 in the future. Please be aware that you must use Arm Compiler 6 for future development because Arm Compiler 5 support will be deprecated in September 2019.</span>
```

Comment:
```suggestion
<span class="notes">**Note:** Arm Compiler 6 is the default Arm Compiler for Mbed OS development, and [is free for use with Mbed OS](https://os.mbed.com/docs/mbed-os/development/tools/index.html#compiler-versions). Most Mbed OS platforms are already compatible with Arm Compiler 6. Some existing targets still supporting Arm Compiler 5 will be migrated to Arm Compiler 6 in the future. Please be aware that you must use Arm Compiler 6 for future development because Arm Compiler 5 support will be deprecated in September 2019.</span>
```

Reply:
@SenRamakri this is a fork, not a branch. I therefore don't have permission to push these suggestions. Please push if you approve them.